cli/src/cli-rpc-ops.c

```diff
@@ -5578,6 +5579,13 @@ cmd_profile_volume_brick_out(dict_t *dict, int count, int interval)
         gf_log("cli", GF_LOG_DEBUG, "failed to get %s from dict", key);
     }
 
+    if (interval >= 0) {
```

Comment:
Why we don't show the fops per second in the cumulative data ?

Reply:
done

Reply:
You have removed this check, but the data is still from an interval right ? shouldn't we compute the accumulated fops/s additionally to the interval data ?